internal/server/deputies.go

```diff
@@ -65,6 +66,16 @@ func listPaAndProDeputyTeams(allTeams []model.Team, requiredTeamTypes []string,
 	return teamsToReturn
 }
 
+func getProTeamIdsAsString(allTeamIds []model.Team) []string {
+	teamIdsToReturn := []string{}
+	for _, tt := range allTeamIds {
+		if tt.Type == "PRO" {
+			teamIdsToReturn = append(teamIdsToReturn, strconv.Itoa(tt.Id))
+		}
+	}
+	return teamIdsToReturn
+}
+
```

Comment:
We want this as a string as it'll be used for the selectedECMs and thats what that works in.